src/autocomplete/chip.css.ts

```diff
@@ -4,21 +4,29 @@ export default css`
 	:host {
 		border-radius: var(--cosmoz-autocomplete-chip-border-radius, 500px);
 		background: var(--cosmoz-autocomplete-chip-bg-color, #cbcfdb);
-		margin: 0px 0 2px 0;
+		margin-bottom: 2px;
 		display: flex;
-		flex-direction: row;
 		align-items: center;
 		flex: 0.0001 1 fit-content;
 		max-width: 18ch;
 		min-width: 40px;
-		padding: 0 4px 0 8px;
+		padding-inline: 8px;
 		gap: 4px;
 		cursor: pointer;
 		transform: translateY(var(--cosmoz-autocomplete-chip-translate-y, 0));
 	}
+
+	:host([data-state='excluded']) {
+		background: var(--cosmoz-autocomplete-excluded-chip-bg-color, #f44336);
+	}
```

Comment:
The chip and listbox should not know anything about `excluded`. We should make sure that all changes specifically for `excluding` are segregated to the `excluding` folder. Only generic changes should be done in the other folders. `data-state='excluded'` is too specific and has no place in the generic chip style.